website/docs/guides/version-6-upgrade.html.markdown

```diff
@@ -0,0 +1,100 @@
+---
+subcategory: ""
+layout: "aws"
+page_title: "Terraform AWS Provider Version 6 Upgrade Guide"
+description: |-
+  Terraform AWS Provider Version 6 Upgrade Guide
+---
+
+# Terraform AWS Provider Version 6 Upgrade Guide
+
+Version 6.0.0 of the AWS provider for Terraform is a major release and includes changes that you need to consider when upgrading. This guide will help with that process and focuses only on changes from version 5.x to version 6.0.0. See the [Version 5 Upgrade Guide](/docs/providers/aws/guides/version-5-upgrade.html) for information on upgrading from 4.x to version 5.0.0.
+
+Upgrade topics:
+
+<!-- TOC depthFrom:2 depthTo:2 -->
+
+- [Provider Version Configuration](#provider-version-configuration)
+- [Provider Arguments](#provider-arguments)
+- [Dropping Support For Amazon SimpleDB](#dropping-support-for-amazon-simpledb)
+- [AWS OpsWorks Stacks End of Life](#aws-opsworks-stacks-end-of-life)
+- [resource/aws_redshift_service_account](#resourceaws_redshift_service_account)
+
+<!-- /TOC -->
+
+## Provider Version Configuration
+
+-> Before upgrading to version 6.0.0, upgrade to the most recent 5.X version of the provider and ensure that your environment successfully runs [`terraform plan`](https://www.terraform.io/docs/commands/plan.html). You should not see changes you don't expect or deprecation notices.
```

Comment:
With the handful of "soft" deprecations we have on resources like `aws_s3_bucket` and `aws_iam_role`, it is still possible to get a deprecation warning with configuration that is still valid in V6. Perhaps we reword this slightly?

```suggestion
-> Before upgrading to version 6.0.0, upgrade to the most recent 5.X version of the provider and ensure that your environment successfully runs [`terraform plan`](https://www.terraform.io/docs/commands/plan.html). You should not see changes you don't expect, or deprecation notices for anything mentioned in this guide.
```